tests/test_main.cpp

```diff
@@ -131,6 +131,11 @@ int test_main(int argc, char *argv[]) {
 		memcpy(doctest_args[x], str, strlen(str) + 1);
 	}
 
+	test_context.setOption("order-by", "name");
+	test_context.setOption("abort-after", 20);
+	test_context.setOption("no-breaks", true);
+
+	// Allow the command-line to override doctest options above.
```

Comment:
This was removed in #43721.

Reply:
Ah, didn't realize I was changing `test_main.cpp`, probably my rebase re-added this.